crates/matrix-sdk/tests/integration/widget.rs

```diff
@@ -601,21 +601,22 @@ async fn send_room_name() {
 }
 
 #[async_test]
-async fn send_future_room_message_event() {
+async fn send_delayed_event_room_message_event() {
```

Comment:
Nitpick, especially for consistency with the following test
```suggestion
async fn send_delayed_message_event() {
```